src/translations/en.json

```diff
@@ -1230,7 +1230,9 @@
           "entities": {
             "entities": "Entities",
             "add_entities_lovelace": "Add to Lovelace",
-            "none": "This device has no entities"
+            "none": "This device has no entities",
+            "hide_disabled": "Hide disabled",
+            "disabled_entities": "+{count} {count, plural,\n  one {disabled entity}\n  other {disabled entities}\n}"
```

Comment:
disabled could be before the plural I think (or some languages should require a different work for disabled for a different plural form)

Reply:
I think I don't understand your suggestion.  Now both words can be translated depending on the plural form. For example in Polish:
```
+1 wyłączona encja
+2 wyłączone encje
+5 wyłączonych encji
```